sampleFiles/sample-schema.json

```diff
@@ -1158,6 +1158,7 @@
               "name": "presenting_symptoms",
               "description": "Indicate presenting symptoms at time of primary diagnosis.",
               "valueType": "string",
+              "isArray": true,
```

Comment:
is this only for code lists ?

Reply:
Not necessarily, I accounted for regex and ranges too since I don't see why they also couldn't be arrays. Right now this is the only field that is going to be an array, but I added some other test cases in the unit tests for other fields types..